### ISSUE TITLE
btrbk-0.22.1 breaks with btrfs-progs < 4.1

### ISSUE THREAD

**Issue by digint** [digint/btrbk#68]
Unfortunately, btrbk-v0.22.1 introduces incompatibilities for btrfs-progs < 4.1.

the commit: 13a656a6d4f965954be397f94cec87d7a0ce6b6b ("add more sophisticated checks for possible future incompatibilities of btrfs-progs") was too restrictive.

Resulting error for `btrbk {run,dryrun}`:
```
ERROR: Failed to parse subvolume detail (unsupported btrfs-progs) for: /path/to/subvol
```

**Comment by digint** [comment]
fixed in: 6f66f72c26461090f1af6e9e754a2ea7c98665a2

**Comment by digint** [comment]
fixed in btrbk-v0.22.2
